_includes/includes/page-version.php

```diff
@@ -1,5 +1,13 @@
 <?php
 
+  // Mappings for files that got renamed at some point. Has to be in
+  // both directions.
+  global $FileMappings;
+  $FileMappings = [
+    'whats-new' => 'whatsnew',
+    'whatsnew' => 'whats-new',
+  ];
```

Comment:
This could go in a separate file. Also given it is a global variable, can we call it `$PageVersionFileMappings`?

And ... I wonder if we could make it easier to maintain -- and support multi-way mappings, by having arrays of mappings:

```suggestion
  // Mappings for files that got renamed at some point. Has to be in
  // both directions.
  global $FileMappings;
  $PageVersionFileMappings = [
    ['whats-new', 'whatsnew'],
  ];
```